### PR TITLE
[#2147] update to asm 9.5

### PR DESCRIPTION
**Pull request by cdietrich**
[#2147] update to asm 9.5

### CODE REVIEW

**Comment by cdietrich** [comment]
@szarnekow @LorenzoBettini any idea what could be the problem here:
https://ci.eclipse.org/xtext/job/xtext/job/cd_asm950/4/

**Comment by LorenzoBettini** [comment]
> @szarnekow @LorenzoBettini any idea what could be the problem here: https://ci.eclipse.org/xtext/job/xtext/job/cd_asm950/4/ the asm it complains to missing seems at least to be in the antDeps dir
> 
> how can i run this test easy locally from maven? how do i need to run the full build and then the build in ide.tests to call this test what do i need to deploy stuff to a local maven repo and then consume it from there?

Sorry, I'm late, but I see you figured it out.
I thought I had left comments on that project documenting how to prepare for running tests from eclipse.

**Comment by cdietrich** [comment]
@LorenzoBettini i was asking for running a single test for maven.
i still have the problen that the branches maven artifacts now sit in my local maven repo
this is why i am asking

File: org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/tp/domainmodel.target [hunk 1]
```diff
@@ -18,6 +18,10 @@
       <unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
       <repository location="https://download.eclipse.org/modeling/tmf/xtext/updates/nightly"/>
     </location>
+    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
```
Comment (cdietrich) [inline org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.releng/tp/domainmodel.target:21]: meh.

**Review by LorenzoBettini — APPROVED** [review]

File: NOTICE.md [hunk 2]
```diff
@@ -86,15 +86,15 @@ Google Guice / Inject Core API (5.0.1)
 
 * License: Apache License, 2.0
 
-Gradle Wrapper (7.4)
+Gradle Wrapper (8.x)
```
Comment (szarnekow) [inline NOTICE.md:89]: Is this still being used?

**Review by szarnekow — APPROVED** [review]

File: NOTICE.md (hunk 2 above)
Comment (LorenzoBettini) [reply NOTICE.md:89]: I think we store it in our resources of the wizard bundle and create it when we generate a project based on Gradle, is that right @cdietrich ?

Comment (cdietrich) [reply NOTICE.md:89]: yes, this is why this is there in the first place

**Pull request merged**